libyul/ControlFlowSideEffectsCollector.cpp

```diff
@@ -51,9 +51,15 @@ void ControlFlowBuilder::operator()(If const& _if)
 {
 	visit(*_if.condition);
 	ControlFlowNode* node = m_currentNode;
-	(*this)(_if.body);
+
+	ControlFlowNode* ifEnd = newNode();
+	node->successors.emplace_back(ifEnd);
+
 	newConnectedNode();
-	node->successors.emplace_back(m_currentNode);
+    (*this)(_if.body);
+	
+    m_currentNode->successors.emplace_back(ifEnd);
+    m_currentNode = ifEnd;
```

Comment:
Bad indent. You used spaces here while we indent with tabs. That's why the coding style CI job is failing.